Translation corrections (#498)
07th-mod/higurashi-console-arcs: Update/kage_006.txt

//rせっかくのお祝いの席を湿っぽくするなんて、無粋も過ぎるというものだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "せっかくのお祝いの席を湿っぽくするなんて、無粋も過ぎるというものだ。",

> 			NULL, "It would be boorish of her to leave her seat all wet when they came all this way out to celebrate.", Line_Normal);

Reviewer: ```suggestion
			NULL, "It would be boorish of her to look gloomy when they came all this way to celebrate.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 28, "ps3/s08/28/320500711", 256, TRUE);
	OutputLine(NULL, "それに警察庁で力になってくれていた人も、今度異動になるみたいだし…」",

> 			NULL, "Plus, the persona at the NPA who'd been helping me out is apparently getting transfered soon, so...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "Plus, the persona at the NPA who'd been helping me out is apparently getting transferred soon, so...\"", Line_Normal);
```

			NULL, "\"...Ah, ahahaha.", Line_WaitForInput);
	ModPlayVoiceLS(4, 28, "ps3/s08/28/320500714", 256, TRUE);
	OutputLine(NULL, "ごめんね、せっかくのめでたい席に変なこと思い出させて。",

> 			NULL, " Sorry for bringing up such awkward memories when we're supposed to be celebrating.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " Sorry for dragging up awkward memories when we're supposed to be celebrating.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#a4856e>まどか</color>", NULL, "<color=#a4856e>Madoka</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 29, "ps3/s08/29/321200169", 256, TRUE);
	OutputLine(NULL, "「だから、そういうところがケチだって…",

> 			NULL, "\"Come on, why are you being stingy when it comes to this...", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Come on, why are you being stingy about this...?", Line_WaitForInput);
```

//rただ、…妹が以前のように、自分以外に味方がいなかった頃の彼女のままではないのだということを悟って、…それが、寂しかった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "ただ、…妹が以前のように、自分以外に味方がいなかった頃の彼女のままではないのだということを悟って、…それが、寂しかった。",

> 			NULL, "Still... realizing that her sister was no longer the girl she'd once known who had no other ally in the world apart from Tomoe... made her feel lonely.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Still... realizing that her sister was no longer the girl she'd once known, who had no other ally in the world apart from Tomoe... made her feel lonely.", Line_Normal);
```

			NULL, " I didn't mean to keep it a secret.", Line_WaitForInput);
	ModPlayVoiceLS(4, 29, "ps3/s08/29/321200179", 256, TRUE);
	OutputLine(NULL, "　ただ、…もう少しちゃんと目処がついたら、報告しようかなって思ってただけで…」",

> 			NULL, " I just... thought it would be better to wait until I was sure things were steady before telling...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " I just... thought it would be better to wait until I was sure things were steady before telling you...\"", Line_Normal);
```

//r今さら時効だと思うが、もしもまどかが家出当時、とある男性の家に転がり込んだことがここでバレようものなら、即刻朝まで説教タイムが開始されるだろう…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "今さら時効だと思うが、もしもまどかが家出当時、とある男性の家に転がり込んだことがここでバレようものなら、即刻朝まで説教タイムが開始されるだろう…。",

> 			NULL, "She knew it would be too late by this point, but if it came out now that Madoka had been living with a man when she ran away from home or something, Tomoe would have started lecturing her until the sun rose...", Line_Normal);

Reviewer: ```suggestion
			NULL, "She knew it would be too late by this point, but if it came out now that Madoka had been living with a man when she ran away from home or something, Tomoe would have started lecturing her until the sun came up...", Line_Normal);
```

			NULL, " And Inspector Akasaka has a daughter of his own.", Line_WaitForInput);
	ModPlayVoiceLS(4, 28, "ps3/s08/28/320500808", 256, TRUE);
	OutputLine(NULL, "…まだ確定じゃない状況で、安易に話すのもどうかと思ったしね」",

> 			NULL, " ...The details aren't certain yet, so I think it would be difficult to discuss it.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " ...We haven't finalized the details yet, so I think it would be difficult to discuss it.\"", Line_Normal);
```

//rやはりこの病院で処方されたものではないということだろう。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "やはりこの病院で処方されたものではないということだろう。",

> 			NULL, "Therefore, this medicine was probably not prescribed at this hospital after all.", Line_Normal);

Reviewer: ```suggestion
			NULL, "So, this medicine probably wasn't prescribed at this hospital after all.", Line_Normal);
```

//rだから、…離れようと思った。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "だから、…離れようと思った。",
			NULL, "So... Madoka thought it would be best if she left.", Line_Normal);
	ClearMessage();

//rそうすることで、姉はきっと自分自身の幸せに対して、少しは目を向けるようになるんじゃないかと考えて…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そうすることで、姉はきっと自分自身の幸せに対して、少しは目を向けるようになるんじゃないかと考えて…。",

> 			NULL, "In doing so, she thought that her sister might eventually focus her own happiness...", Line_Normal);

Reviewer: ```suggestion
			NULL, "In doing so, she thought that her sister might eventually focus on her own happiness...", Line_Normal);
```

//r巴にはずいぶんとバカ呼ばわりさせて恨まれたが、…これでよかったのだと、まどかは今でも胸を張って言うことができた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "巴にはずいぶんとバカ呼ばわりさせて恨まれたが、…これでよかったのだと、まどかは今でも胸を張って言うことができた。",

> 			NULL, "Tomoe still treated her like an idiot and had her resentments... but even now, Madoka could proudly say she was okay with that.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Tomoe still treated her like an idiot and felt some resentment... but even now, Madoka could proudly say she was okay with that.", Line_Normal);
```
